collectors/o365/lib/o365_mgmnt/index.js

```diff
@@ -57,8 +57,8 @@ var _listSubscriptions = function() {
  * @returns {Promise}
  *
  */
-var _startSubscription = function(contentType) {
-    return getO365ManagmentClient().then((client) => {
+var _startSubscription = function(contentType, pawsCreds) {
```

Comment:
can we please have `pawsCreds` as the first argument in every function

Reply:
actually, please rename `pawsCreds` to just `creds` or `azureCreds`

Reply:
used creds as first parameter